### PR TITLE
skr-without-sc-integration-dev

### PR DESCRIPTION
**Pull request by tillknuesting**
Change test to support provisioning and test after that without SC

https://github.com/kyma-project/kyma/issues/13415

### CODE REVIEW

**Comment by tillknuesting** [comment]
(keep in mind that test might fail as they are executed against pipeline with SC)

**Review by wozniakjan — APPROVED** [review]
/lgtm
/approve

the failures in test executions appear to be unrelated to the changes introduced here
```
  2 failing

  1) Execute SKR test
       CommerceMockTest()
         function should be reachable through secured API Rule:
     AssertionError: expected { Object (event, podName) } to have nested property 'order.totalPriceWithTax.value'
      at checkFunctionResponse (test/fixtures/commerce-mock/index.js:144:35)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
      at async Context.<anonymous> (skr-test-without-sc/skr-test.js:139:7)

  2) Execute SKR test
       CommerceMockTest()
         Amount of audit events must not exceed a certain threshold:
     TypeError: Cannot read property 'value' of undefined
      at checkAuditEventsThreshold (audit-log/helpers.js:63:40)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
      at async Context.<anonymous> (skr-test-without-sc/skr-test.js:165:9)
```

File: tests/fast-integration/skr-test-without-sc/skr-test.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+          false,
+          true,
+      );
+    });
+
+    it('function should be reachable through secured API Rule', async function() {
+      await checkFunctionResponse(this.options.testNS);
+    });
+
+    // TODO: LegacyEvent needed after not SC is present?
```
[the remark this answers is not in the document]
Comment (wozniakjan) [reply tests/fast-integration/skr-test-without-sc/skr-test.js]: Thanks, adding that back

File: tests/fast-integration/skr-test-without-sc/readme.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+```
+
+Run the test scenario:
+
+```bash
+npm run skr-test-without-sc
+```
+
+## Environment variables
+`AL_SERVICE_KEY` must be a cloud foundry service key with the info about `UAA (User Account and Authentication)`. Learn more about [Managing Service Keys in Cloud Foundry](https://docs.cloudfoundry.org/devguide/services/service-keys.html).
```
Comment (grego952) [inline tests/fast-integration/skr-test-without-sc/readme.md:20]:
```suggestion
`AL_SERVICE_KEY` must be a Cloud Foundry service key with the `UAA (User Account and Authentication)` information. To learn more, read about [managing service keys in Cloud Foundry](https://docs.cloudfoundry.org/devguide/services/service-keys.html).
```

**Review by grego952 — APPROVED** [review]

**Review by wozniakjan — APPROVED** [review]
/lgtm
/approve

**Review by Tomasz-Smelcerz-SAP — APPROVED** [review]

**Review by PK85 — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**